superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -246,7 +254,8 @@ export default function ActivityTable({
         );
       },
     );
-  if (loadingState && !editedObjs) {
+
+  if ((loadingState && !editedObjs) || loadedCount !== 3) {
```

Comment:
I would use `<` for this check just to be safe. But actually, unless the number itself is important, I would probably separate the loading states into three different booleans and `&&` them together.

Reply:
Yep makes sense